app/tests/e2e/helpers.js

```diff
@@ -132,6 +132,7 @@ async function actionModalCheckout(
     ".action-modal-footer .button:nth-of-type(2):enabled",
     30000
   )
+  await browser.pause(500)
```

Comment:
what are we waiting for here?

Reply:
Test stops at this point, I'm still testing but seems some query is not loade. Then when click on the Next button it doesn't do anything and therefore you don't get the invoice.

Reply:
Can we wait explicit for the balances to have been loaded?

Reply:
Great @faboweb , that pause was not the best way to fix the problem. Disabling next button until balances loaded in action modal is a cleaner way so far.